packages/fxa-auth-server/lib/senders/emails/partials/brandMessaging/index.txt

```diff
@@ -0,0 +1,6 @@
+<% if (locals.brandMessagingMode == 'postlaunch') { %>
+brand-banner-message = "Did you know we changed our name from Firefox accounts to Mozilla accounts? Learn more"
+
+https://support.mozilla.org/en-US/kb/firefox-accounts-renamed-mozilla-accounts
```

Comment:
```suggestion
https://support.mozilla.org/kb/firefox-accounts-renamed-mozilla-accounts
```

We should remove the locale from the URL so it will jump to the correct locale based on user's language preferences.